spec/models/book_spec.rb

```diff
@@ -48,9 +47,9 @@
   end
 
   describe '.add_task' do
-    before { Book.add_task('0804720991') }
     it 'adds a dirty book when given an isbn' do
-      expect(Book.where(isbn_10: '0804720991', dirty: true).exists?).to be_truthy
+      expect(SyncBookJob).to receive(:perform_later).once
+      Book.add_task('0804720991')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.